wagtail/users/views/bulk_actions/user_bulk_action.py

```diff
@@ -11,7 +11,7 @@ def get_all_objects_in_listing_query(self, parent_id):
         listing_objects = self.model.objects.all().values_list("pk", flat=True)
         if "q" in self.request.GET:
             q = self.request.GET.get("q")
-            model_fields = [f.name for f in self.model._meta.get_fields()]
+            model_fields = frozenset(f.name for f in self.model._meta.get_fields())
```

Comment:
suggestion: add a note as to why frozenset is used

Reply:
I'm personally a fan of just using `set` unless it's absolutely necessary. Reading `frozenset` is likely to make someone question why it needs to be frozen - which it doesn't appear to need to be in this case.

On a similar note, why _is_ this changing to a `set`?

Reply:
> On a similar note, why is this changing to a set?

It was defined as a list while the only operation we use it for is looking up names. The list seems to contain one to two dozen elements so I feel using a set/frozenset makes more sense.

> why it needs to be frozen

It doesn't need to be frozen. However, I prefer using it since it's immutable.

Reply:
I've changed `frozenset` to `set`.